frontend/widgets/OBSBasic_Scenes.cpp

```diff
@@ -37,6 +37,19 @@ template<typename OBSRef> struct SignalContainer {
 	OBSRef ref;
 	vector<shared_ptr<OBSSignal>> handlers;
 };
+
+QDataStream &operator<<(QDataStream &out, const SignalContainer<OBSScene> &v)
+{
+	out << v.ref;
+	return out;
+}
+
+QDataStream &operator>>(QDataStream &in, SignalContainer<OBSScene> &v)
+{
+	in >> v.ref;
+	return in;
+}
+
```

Comment:
I suppose we can't put these into `shared/qt/wrappers/qt-wrappers.cpp` with the other `QDataStream &operator` definitions?

Reply:
In C++, function definitions are part of the API of the class they specialise for by its types.

So the operators here belong to the `SignalContainer` type (and not the `QDataStream` type).